### DISCUSSION TITLE
[BUG] Longhorn shows more capacity than is actually in the filesystem directory.

### DISCUSSION THREAD

**Discussion by bautistamad** [longhorn/longhorn#5226] [General]
I am using nats which is deployed with helm with 3 replicas of it, each of these replicas contains its own PVC with 10Gb of storage, the problem is when I enter the longhorn interface it shows me more capacity than there really is.

Storage of PVC:
![image](https://user-images.githubusercontent.com/75149705/211037349-1fa8dff2-c7a1-4256-8107-7ccbbf7ef7aa.png)

Longhorn UI storage:
![image](https://user-images.githubusercontent.com/75149705/211037796-4fcf5235-8460-46ab-a73b-1a26511a4993.png)

 - Longhorn version: 1.3.2
 - Installation method (e.g. Rancher Catalog App/Helm/Kubectl): Helm
 - Kubernetes distro (e.g. RKE/K3s/EKS/OpenShift) and version: RKE
   - Number of management node in the cluster: 1
   - Number of worker node in the cluster: 4

**Comment by derekbit** [comment]
Please refer to https://longhorn.io/docs/1.4.0/volumes-and-nodes/volume-size. Thanks.
